cf_test.go

```diff
@@ -192,23 +58,15 @@ func TestGetOrganizationName(t *testing.T) {
 					organizationName: "org-1",
 					organizationGuid: "guid-1",
 				},
-				ServiceInstances: &mockServiceInstances{},
-				ServiceOfferings: &mockServiceOfferings{},
-				ServicePlans:     &mockServicePlans{},
-				Spaces:           &mockSpaces{},
+				Spaces: &mockSpaces{},
```

Comment:
Update the indentation between `Spaces:` and `&mockSpaces{},` to be the same on lines 61 and 69?

Reply:
Nevermind, looking at the file more closely the formatting makes sense

Reply:
This indentation is actually what is generated by `gofmt`. I'm not sure what rules it follows, but apparently this syntax matches Go conventions